djstripe/models/core.py

```diff
@@ -711,12 +711,18 @@ def subscribe(
             trial_end=trial_end,
             trial_from_plan=trial_from_plan,
             trial_period_days=trial_period_days,
+            expand=["latest_invoice.payment_intent"], #XXX Hotfix for https://stripe.com/docs/billing/subscriptions/set-up-subscription#manage-sub-status
         )
 
+        #XXX Hotfix for https://stripe.com/docs/billing/subscriptions/set-up-subscription#manage-sub-status
+        if 'latest_invoice' in stripe_subscription and 'payment_intent' in stripe_subscription['latest_invoice']:
+            status = stripe_subscription['latest_invoice']['payment_intent']['status']
+            client_secret = stripe_subscription['latest_invoice']['payment_intent']['client_secret']
+
         if charge_immediately:
             self.send_invoice()
 
-        return Subscription.sync_from_stripe_data(stripe_subscription)
+        return (Subscription.sync_from_stripe_data(stripe_subscription), status, client_secret)
```

Comment:
This is a breaking change since you're changing the return type to a tuple.

Reply:
Ah! I am so sorry that I hadn't ran the test before creating this PR.